getresolution.js

```diff
@@ -0,0 +1,8 @@
+
+function getResolution(){
```

Comment:
Great initiative, Matthias! Like your increasing number of contributions ;)

What is the issue you're trying to fix here? Is this related to a issue? If not, please create an issue describing the details resulting in you creating this PR.

Two things regarding this function:
1) Should this maybe be added to index.js instead of kept in a separate file?
2) There is a typo in the return statement, meaning the variable gets never returned

Reply:
In case "https://images.unsplash.com/" is down , the brokenImage function gets executed and shows the picture from the repo. 

I think it could be necessary to get the screen resolution. However I did not use this getResolution function at all. Probably might be useful if someone is viewing the page from a real small screen and the code makes use of it.